### PR TITLE
[4.0] Option tabs descriptions

### PR DESCRIPTION
**Pull request by brianteeman**
Somewhere along the way we lost the display of the fieldset descriptions in the component options. This puts it back. There may be other accessibility issues with this view but it is beyond the scope of this PR

### Before
image

### After
image

### CODE REVIEW

File: administrator/components/com_config/tmpl/component/default.php [hunk 1]
```diff
@@ -66,6 +66,11 @@
 			<div class="tab-content" id="configContent">
 				<?php foreach ($this->fieldsets as $name => $fieldSet) : ?>
 					<div class="tab-pane" id="<?php echo $name; ?>">
+						<?php if (isset($fieldSet->description) && !empty($fieldSet->description)) : ?>
```
Comment (Quy) [inline administrator/components/com_config/tmpl/component/default.php]:
```suggestion
						<?php if (!empty($fieldSet->description)) : ?>
```

Comment (Quy) [reply administrator/components/com_config/tmpl/component/default.php]: `isset` is included in the `empty` test.

Comment (brianteeman) [reply administrator/components/com_config/tmpl/component/default.php]: It is a direct copy from joomla 3

Comment (Quy) [reply administrator/components/com_config/tmpl/component/default.php]: Let's improve it.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e944eb60b9d6c9e9c2406f6ef556c6cd676cc3c0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24984">issues.joomla.org/tracker/joomla-cms/24984</a>.</sub>

**Comment by brianteeman** [comment]
Updating branch to trigger drone

**Comment by brianteeman** [comment]
Updated branch to trigger drone again

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**